### ISSUE TITLE
Sommelier on Pixelbook is too big.

### ISSUE THREAD

**Issue by GioPanaro** [skycocker/chromebrew#3447]
## Before you submit an issue
Please look up our issues, maybe your problem was mentioned before.

## Description
Windows opened in sommelier on the PixelBook are far too large. Changing the scale in sommelier.env fixes the issue but shrinks the mouse down to microscopic size

If you executed a specific command:
Start any application with a GUI such as gimp on the Pixelbook

## Versions and system information
chromebrew: `crew --version`
1.2.4

ChromeOS:
 ```CHROMEOS_ARC_ANDROID_SDK_VERSION=28
CHROMEOS_ARC_VERSION=5829310
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_RELEASE_BOARD=eve-signed-mp-v2keys
CHROMEOS_RELEASE_BRANCH_NUMBER=0
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R78-12457.0.0
CHROMEOS_RELEASE_BUILD_NUMBER=12457
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=78
CHROMEOS_RELEASE_DESCRIPTION=12457.0.0 (Official Build) canary-channel eve 
CHROMEOS_RELEASE_KEYSET=mp-v2
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=canary-channel
CHROMEOS_RELEASE_VERSION=12457.0.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=12457.0.0
```

Model: Pixelbook

Architecture: `x86_64`

## Addtional information
Mention things we might need to know. Like:

The scale in sommelier.env is set to 2 on my Pixelbook which ends up being a bit blown up still, but fine. The mouse pointer appears to be 1/4 the size in such windows.

---

## That's it
Thank you for submitting your issue.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.

**Comment by uberhacker** [comment]
@GioPanaro: What specific app is causing your scaling issue?  If the app depends on gtk2 or gtk3, there is a good chance you can configure settings in `~/.gtkrc-2.0` and `~/.config/gtk-3.0/settings.ini` respectively.

**Comment by JL2210** [comment]
This is [in the FAQ](https://github.com/skycocker/chromebrew/wiki/FAQ). Scroll down to "Q: I'm having troubles with font size in apps depending on GTK2. What do I do?" and you should see it.

**Comment by JosephSamela** [comment]
Found this ticket searching for a solution to this same problem!

I installed codium (vscode) and everything (text, icons, menus) was blurry. I increased `SCALE` from `1` to `2` and the blurriness went away - but now the cursor is small when over the window - it's regular size when over another window.

`~/.sommelier.env` file looks like this.

```
GDK_BACKEND=x11
CLUTTER_BACKEND=wayland
XDG_RUNTIME_DIR=/var/run/chrome
WAYLAND_DISPLAY=wayland-0
DISPLAY=:0
SCALE=2
```

**Comment by Muvuk** [comment]
Helo Guys, 

Thank you for chromebrew, it's just soo good. 

I have a similar issue this is why I comment here instead of opening new issue. 

As you can see on the screenshots, something wrong with the window/content size. And what I found in the faq just did not solve the issue. 

Do you have any idea how to fix this? (it worked in the past, but I had to reinstall chromebrew bc of this issue:
https://www.reddit.com/r/chromeos/comments/sn6bsv/chrome_os_98_is_rolling_out/hw3osax/

![Screenshot 2022-02-15 12 28 46](https://user-images.githubusercontent.com/5338127/154053923-026567cd-e32d-416c-a626-fabf981c7069.png)
![Screenshot 2022-02-15 12 28 41](https://user-images.githubusercontent.com/5338127/154053927-ae1a6cad-560a-4dd6-87ac-7a25240d783e.png)

**Comment by Muvuk** [comment]
It works! Thank you
